src/Mod/Sketcher/Gui/TaskSketcherElements.cpp

```diff
@@ -61,29 +61,29 @@ using namespace Gui::TaskView;
 
 // Translation block for context menu: do not remove
 #if 0
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Point Coincidence");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Point on Object");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Vertical Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Horizontal Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Parallel Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Perpendicular Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Tangent Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Equal Length");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Symmetric");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Block Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Lock Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Horizontal Distance");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Vertical Distance");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Length Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Radius Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Diameter Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Radiam Constraint");
-QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Angle Constraint");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Point coincidence");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Point on object");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Vertical constraint");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Horizontal constraint");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Parallel constraint");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Perpendicular constraint");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Tangent constraint");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Equal constraint");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Symmetric constraint");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Block constraint");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Lock");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Horizontal dimension");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Vertical dimension");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Length dimension");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Radius");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Diameter");
+QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Radius/diameter");

```

Comment:
```suggestion
QT_TRANSLATE_NOOP("SketcherGui::ElementView", "Radius or diameter");
```
Not sure if it should say "dimension" after it. Same for "Radius, "Diameter" and "Angle"